app/components/rate-goal/rate-goal.jsx

```diff
@@ -2,15 +2,24 @@ import React from 'react';
 import GoalTileComponent from '../goal-tile.jsx';
 import RateGoalSlider from './rate-goal-slider.jsx';
 
+import Cactus from './animations/cactus.jsx';
+
+const RateGoal = ({ currentGoal, onMoveSlider, feedback, setPreviousScore, }) => {
+
+  const animation = <Cactus
```

Comment:
we do this because we want to have alternative animations, i.e. won't always be a cactus